app/serializers/api/v1/customer_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class CustomerSerializer < BaseSerializer
+      attributes :id, :enterprise_id, :first_name, :last_name, :code, :email
```

Comment:
Reading the initial issue: https://github.com/openfoodfoundation/openfoodnetwork/issues/8788, I was wondering if there were some necessary attributes missing?

Reply:
This is not closing #8788. This is only step two of #8787. I will create another PR for the missing attributes and the feature toggle.

Reply:
oh yes, I didn't see that step2 ! Thanks !